guides/common/assembly_installing-server-connected.adoc

```diff
@@ -10,6 +10,7 @@ ifdef::satellite[]
 include::modules/proc_registering-to-red-hat-subscription-management.adoc[leveloffset=+1]
 endif::[]
 
+// TODO: evalute the structure of this assembly until line 22
```

Comment:
Would it be better to file Issues instead of adding TODO and DISCUSS-ME comments?
The comments will get lost... :)

Reply:
I made some notes for myself about this.